### PR TITLE
chore(ENG-3452): dockerhub updates

### PR DESCRIPTION
**Pull request by mstrisoline**
## Description

- updates to work with dockerhub

## Testing required outside of automated testing?
will need merged to test
- [ ] Not Applicable

### Screenshots (if appropriate):

- [ ] Not Applicable

### CODE REVIEW

File: .github/workflows/publish-and-deploy.yml [hunk 1]
```diff
@@ -38,8 +38,8 @@ jobs:
       - name: Build
         uses: whoan/docker-build-with-cache-action@v5
         with:
-          context: basistheory/${{ matrix.demo }}
-          image_name: ${{ matrix.demo }}
+          context: demos/${{ matrix.demo }}
+          image_name: basistheory/${{ matrix.demo }}
```
Comment (djejaquino) [inline .github/workflows/publish-and-deploy.yml:42]: Is this acceptable?
```suggestion
          image_name: basistheory/demos/${{ matrix.demo }}
```

Comment (mstrisoline) [reply .github/workflows/publish-and-deploy.yml:42]: We can try it, but i believe its `org/container:tag` in dockerhub for the syntax. All the examples I am seeing are in the previously noted format.

Comment (mstrisoline) [reply .github/workflows/publish-and-deploy.yml:42]:
```
Your repository name must contain a combination of alphanumeric characters and may contain the special characters ., _, or -. Letters must be lowercase.
```
^what dockerhub gives me if I try to do `demos/reference-check` i can rename the repository to `demo-reference-check` if you want more contextual clarity in the name / purpose of the container. There is also a description section I was going to fill out and connect everything back to the github repo.

Comment (djejaquino) [reply .github/workflows/publish-and-deploy.yml:42]: `reference-check-demo` should suffice. Thanks

**Review by djejaquino — APPROVED** [review]

**Pull request merged**